sync/awsdatabricks.py

```diff
@@ -209,11 +219,14 @@ def _get_cluster_report(
         ]
     )
     if not instances["Reservations"]:
-        return Response(
-            error=DatabricksError(
-                message=f"Unable to find any active or recently terminated instances for cluster `{cluster_id}` in `{aws_region_name}`"
-            )
+        no_instances_message = (
+            f"Unable to find any active or recently terminated instances for cluster `{cluster_id}` in `{aws_region_name}`. "
+            + "Please refer to the following documentation for options on how to address this - <some link>"
```

Comment:
This message will need to be updated to point to the correct documentation page. For now I've left a placeholder in.

Reply:
Probably better not to log TODOs (some link). It could be there for a while.

Reply:
Yeah - let me point to the CLI docs for now, and update this when we have some more comprehensive documentation that goes over some different use cases.